druid/examples/value_formatting/src/formatters.rs

```diff
@@ -173,7 +174,7 @@ impl Formatter<f64> for NaiveCurrencyFormatter {
                 let bad_char = if c1.is_ascii_digit() { other } else { c1 };
                 Validation::failure(CurrencyValidationError::InvalidChar(bad_char))
             }
-            _ => unreachable!(),
+            other => panic!("unexpected: {:?}", other),
```

Comment:
Why `other` vs `_`? Just curious.

Reply:
Just so that I had a simple binding I could print to see what the failure was. Probably don't need to commit, but seems more useful anyway..

Reply:
ohh haha I see now, I didnt notice you use it too.